### DISCUSSION TITLE
Should it be possible to enter future years as representation?

### DISCUSSION THREAD

**Discussion by AlexEhn** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#120] [Q&A] [answered]
I don't see any reason for not being able to enter a year of representation that is later than current year. 
Also if possible the module registry does not need to be updated every year with an updated representation year. 

![image](https://user-images.githubusercontent.com/57874863/131720664-6f0bd329-3d2e-444a-b105-22093b40d53d.png)

**Comment by fjallemark** [comment, marked as answer]
Neither **Represents from year** and **Represents to year** is mandatory.
- If you leave **Represents from year** empty, it means before 1900.
- If you leave **Represents to year** empty, it means *until further notice*.

I can see this documentation is missing in the help text, and it will be added in the next release.

**Reply by AlexEhn** [reply]
Aha, I see. Make sense.
